setup.py

```diff
@@ -37,6 +37,7 @@
         "Topic :: Software Development :: Libraries",
     ],
     packages=find_packages(exclude=["tests", "tests.*", "examples"]),
+    package_data={"": ["py.typed"]},
```

Comment:
```suggestion
    package_data={"questionary": ["py.typed"]},
```
I think this should mention the package, shouldn't it?

Reply:
`""` means from any package in `packages`.  I omitted the name of the package because questionary is detected automatically by `find_packages` but I can change it to`"questionary"` if you prefer.

Reply:
ah great to know, I wasn't aware of that - let's make it explicit though 🙂